travis/docker_run.sh

```diff
@@ -12,4 +12,5 @@ docker run -it \
           -e JEKYLL_ENV=${JEKYLL_ENV} \
           -e TRAVIS_REPO_SLUG=${TRAVIS_REPO_SLUG%/*} \
           -e TRAVIS_EVENT_TYPE=${TRAVIS_EVENT_TYPE} \
+          -e PAT=${PAT} \
```

Comment:
would rather only add pat if its set so maybe something like
```
PAT_ENV=""
if [ -n "$PAT" ]; then
   PAT_ENV="-e $PAT"
fi
```

then in docker run
```
..........
          -e TRAVIS_EVENT_TYPE=${TRAVIS_EVENT_TYPE} \
          ${PAT_ENV}